### ISSUE TITLE
"Sign in to confirm your age"

### ISSUE THREAD

**Issue by cgorlla** [yuliskov/SmartTubeNext#880]
[This](https://youtu.be/EE-4GvjKcfs) video will play absolutely fine on the official YouTube app signed in with the same account, but when trying to play it on the latest version of SmartTube Next, ![20210910_133935.jpg](https://user-images.githubusercontent.com/18127268/132915387-bf6c13c2-8f9d-4ca8-bb8a-579908d807ff.jpg) is the result.

**Comment by esonique** [comment]
I have also been having this issue this. The same videos are playing fine from YouTube app.

![Screenshot_20210925-180302](https://user-images.githubusercontent.com/85214149/134789831-f6f73b96-f5c9-4a13-bf18-9011519c76d6.png)

**Comment by esonique** [comment]
I just noticed both screenshots are off is attempting to play playstation and Xbox videos.

**Comment by KarkanAlzwayed** [comment]
I don't think this is an issue, I think it is a feature they'll need to add. I have the same thing happen every time I play an "age restricted" video. Let's hope they take a look at this.

**Comment by DerBunman** [comment]
I also have the same problem. It happens on all age restricted videos.

**Comment by tsumeone** [comment]
Same problem here on basically all the videos on police activity channel. Used to work fine, not sure what changed. But I am signed in. Works fine signed into official YouTube app.

**Comment by jessgoldq4** [comment]
Still an issue as of 2024

**Comment by dnfm** [comment]
This is still an issue now and I find myself wondering if this is another crappy attempt by Google to block smart tube. Lots of videos started doing this to me today that weren't just a couple days ago.  (Music videos that I'm pretty sure are unrelated [different artists and labels])

**Comment by TrippyPrimate** [comment]
Same anyone find a work around yet?

**Comment by Nororibeqa** [comment]
![Image](https://github.com/user-attachments/assets/fba75486-f84f-4c4f-9921-244b944df5a8) I got the same problem about 3 days ago and also in the Smart Tube Next 26.75 release there is this issue where the Volume now is really low even using the Volume configuration to turn the Volume up to 200% the Volume is still lower than the previous release. Can you fix that please?

**Comment by Nororibeqa** [comment]
Now the Volume is lower than previous releases even turning the volume up using the built in configuration to turn it up
